examples/visualization/visualizers/keypoints.py

```diff
@@ -1,12 +1,12 @@
 import cv2
-import depthai as dai
+import numpy as np
 
 from depthai_nodes.ml.messages import Keypoints
 
 from .utils.message_parsers import parse_keypoints_message
 
 
-def visualize_keypoints(frame: dai.ImgFrame, message: Keypoints, extraParams: dict):
+def visualize_keypoints(frame: np.ndarray, message: Keypoints, extraParams: dict):
```

Comment:
There are other places where this type is incorrectly set. I belieeve this is the case for every `visualize...` function in the `examples/`. Let's correct the types for all of them in this PR.

Reply:
Fixed in [439f4f0](https://github.com/luxonis/depthai-nodes/pull/127/commits/439f4f0cd768189f643bd5cdf7c05f8cf981adbb).